spec/src/main/doc/spi.asciidoc

```diff
@@ -241,9 +241,7 @@ public interface CDIProvider {
 }
 ----
 
-A Java EE container is required to provide a CDI provider that will allow access to the current container for any Java EE application or Java EE module which contains enabled beans.
-
-Java EE components may obtain an instance of +BeanManager+ from JNDI by looking up the name +java:comp/BeanManager+.
+Components may obtain an instance of +BeanManager+ from JNDI by looking up the name +java:comp/BeanManager+.
```

Comment:
This should stay as "Java EE components" and the entire sentence should be moved to the Java EE part